support-logs/riff-raff.yaml

```diff
@@ -6,9 +6,6 @@ deployments:
     type: cloud-formation
     parameters:
       templatePath: cfn.yaml
-      amiParametersToTags:
-        NginxAMI:
-          BuiltBy: amigo
-          AmigoStage: PROD
-          Recipe: ubuntu-xenial-with-nginx
-      amiEncrypted: true
```

Comment:
I'm not sure what data is being logged to / stored on this instance, but you can request an encrypted copy via https://amigo.gutools.co.uk/recipes/ubuntu-xenial-with-nginx/edit if you'd prefer to use an encrypted volume (generally I think that's preferred if there is any personal data involved)?

Reply:
oohhh interesting! It does look quite easy to make it available. I'll give it a try

Reply:
Using encryption increases the complexity of this a bit, as it needs to be set up to use KMS. 

I'd rather go the other way and not store any data to begin with. The only thing that stores data on this box is nginx's access and error logs, which I've just pushed a commit to disable!

Fun fact: there's a lot of misleading advice on the internet that to disable nginx logs you can do `access_log off;`, when it reality, that stores the logs in a file called `off`!